### PR TITLE
docs: first lol ppp

### LINKED ISSUE #12: Create PPP for the first iteration
Write up and publish the PPP (Plan, Progress, Problems) as weekly report to https://makers.bolt.fun/project/10101

### Example PPPs
- https://makers.bolt.fun/story/ppps-week-1-lightsats--207
- https://makers.bolt.fun/story/ppps-week-1--160

### PR DESCRIPTION
**Pull request by holzeis**
resolves #12

### CODE REVIEW

File: ppp/iteration_1_18102022.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Plans 📆
+
+*   Decide on the mobile technology stack.
+*   Setup mobile wallet project.
```
Comment (bonomat) [inline ppp/iteration_1_18102022.md:4]: At this stage we do not have any lightning integration, let's be precise that this is an on-chain wallet.
```suggestion
*   Setup on-chain mobile wallet project.
```

File: ppp/iteration_1_18102022.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+*   Design wireframes to illustrate the user flow and experience.
+    
+
+### Progress ✅
+
+*   Discussed the projects vision and next steps.
+*   Setup github repository.
```
Comment (bonomat) [inline ppp/iteration_1_18102022.md:11]:
```suggestion
*   Setup github repository: https://github.com/itchysats/10101
```

**Review by bonomat — APPROVED** [review]
Thanks for writing this up. Let's go 🚀🌕

File: ppp/iteration_1_18102022.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+### Plans 📆
```
[the remark this answers is not in the document]
Comment (holzeis) [reply ppp/iteration_1_18102022.md:1]: You added a link to the vision in the links. I guess that's sufficient.

**Pull request merged**